src/server/controllers/user.js

```diff
@@ -0,0 +1,83 @@
+const { prisma } = require('../utils/prisma');
+const { SERVER_ERROR, SERVER_SUCCESS, PRISMA_ERROR, KEYS } = require('../config.js');
+
+const { hashedPassword, checkPassword, removeKeys } = require('../utils/auth.js');
+
+const createUser = async (req, res) => {
+	const { username, password, email } = req.body;
+	
+	const passwordHashed = await hashedPassword(password);
+
+	const user = {
+		username,
+		password: passwordHashed,
+		email,
+	};
+
+	try {
+		let createdUser = await prisma.user.create({
+			data: {
+				...user,
+			},
+		});
+
+		createdUser = removeKeys(createdUser, KEYS.PASSWORD);
```

Comment:
Although I appreciate what you're doing here (I especially like you using ... on the parameter in this removeKeys function), I think this is over-engineering a bit.

It's not bad, it just feels a bit like over-kill since you're only removing one property on the object

Reply:
My bad @vherus. I requested this